package.json

```diff
@@ -1,17 +1,17 @@
 {
   "private": true,
   "devDependencies": {
-    "es-check": "^5.2.3",
-    "eslint": "^7.26.0",
+    "es-check": "^5.2.4",
+    "eslint": "^7.29.0",
     "eslint-config-google": "^0.14.0",
-    "source-map": "0.5.6",
+    "source-map": "^0.5.7",
     "ws": "~0.4.28"
   },
   "dependencies": {
-    "acorn": "8.4.1",
-    "google-closure-compiler": "20200920.0.0",
-    "html-minifier-terser": "5.0.2",
-    "wasm2c": "1.0.0"
+    "acorn": "^8.4.1",
+    "google-closure-compiler": "^20200920.0.0",
+    "html-minifier-terser": "^5.1.1",
+    "wasm2c": "^1.0.0"
```

Comment:
Is this safe to do, to start allowing new versions automatically?

Reply:
I think its reasonable since it won't allow new versions in the places we care about such as CI and emsdk.   Those versions are all dictated by the package-lock file.

My understanding is that if you run `npm ci` you always get the specific lock file contents, but if you run `npm install` it is flexible.

Do you think its safer to remove these carets?    I guess maybe it makes sense, but it just means that we need to manually edit the file in order to get `npm install --save` to do anything.

Reply:
Oops, I meant to make that a separate PR anyway.   Updated this PR to be just an update.